src/mmw/js/src/analyze/models.js

```diff
@@ -54,27 +54,37 @@ var AnalyzeTaskModel = coreModels.TaskModel.extend({
     }
 });
 
-AnalyzeTaskModel.getSingleton = function(App, aoi) {
-    if (!App.analyzeModel) {
-        App.analyzeModel = createTaskModel(JSON.stringify(aoi));
+var AnalyzeTaskCollection = Backbone.Collection.extend({
+    model: AnalyzeTaskModel
+});
+
+AnalyzeTaskCollection.getSingleton = function(App, aoi) {
+    if (!App.analyzeCollection) {
+        App.analyzeCollection = new AnalyzeTaskCollection([
+            { area_of_interest: aoi, taskName: 'analyze' },
+            { area_of_interest: aoi, taskName: 'analyze/animals' },
+            { area_of_interest: aoi, taskName: 'analyze/pointsource' },
+        ]);
     }
 
-    return App.analyzeModel;
+    return App.analyzeCollection;
 };
 
-// Pass in the serialized Area of Interest for
-// caching purposes (_.memoize returns the same
-// results for any object), and deserialize
-// the AoI for use on the model.
-var createTaskModel = _.memoize(function(aoi) {
-    return new AnalyzeTaskModel({
-        area_of_interest: JSON.parse(aoi)
-    });
-});
+function createAnalyzeResultViewModelCollection(App, aoi) {
```

Comment:
Sounds like there is a flaw in our dependency chain. One would expect that `App` should be able to include models and collections since it is a high level controller.